deploy/examples/common-second-cluster.yaml

```diff
@@ -21,7 +21,7 @@ roleRef:
 subjects:
```

Comment:
I wonder if the `$NAMESPACE` env var could be converted to also use the same comment pattern `# namespace:cluster` to indicate the cluster. Maybe it's something to consider for a separate PR.

Reply:
@travisn You mean only the first one right:? something like:

```
apiVersion: v1
kind: Namespace
metadata:
  name: $NAMESPACE # namespace:cluster

```

Reply:
In the doc on [adding a second cluster](https://rook.io/docs/rook/latest/Storage-Configuration/Advanced/ceph-configuration/#deploying-a-second-cluster), this yaml is used with the env var substituted. It seems odd that we substitute the cluster namespace with the `NAMESPACE` env var, but then hard-code the operator namespace. Instead of the env var, seems like we should use the same pattern as all other files and replace them with a similar script to the alternate namespace script with the `# namespace:cluster` or `# namespace:operator` comments.